apps/docs/src/app/core/component-docs/dialog/examples/dialog-backdrop-container/dialog-backdrop-container-example.component.ts

```diff
@@ -7,6 +7,15 @@ import { DialogService } from '@fundamental-ngx/core/dialog';
     encapsulation: ViewEncapsulation.None,
     styles: [
         `
+            .fd-button {
```

Comment:
Since you have ViewEncapsulation.None, these styles will also affect any other fd-button on other pages. You can add unique class name to the buttons and style them instead, or, remove ViewEncapsulation.None